samigo/samigo-app/src/webapp/jsf/evaluation/gradeStudentResult.jsp

```diff
@@ -134,7 +134,7 @@ function toPoint(id)
       <f:actionListener
         type="org.sakaiproject.tool.assessment.ui.listener.evaluation.QuestionScoreListener" />
     </h:commandLink>
-    <h:outputText value="</span><li role='menuitem'><span>" escape="false" rendered="#{totalScores.firstItem ne ''}" /> 
+    <h:outputText value="</span><li role='menuitem'><span>" escape="false" rendered="#{totalScores.firstItem ne '' && !totalScores.hasRandomDrawPart}" /> 
```

Comment:
The last closing has to be there to close the last menu item whether it's this one or the one before it.

This whole section is kind of hard to read. Looking at this again, I realized none of the li tags are even closed except for the last one. And it's not just a problem on this page, but on all the evaluation pages with these menu items. I think I'll just clean up this section a bit and see if I can make it reusable for all the other pages that create these menu items.